documents/TestSuite/util/constant_color2.osl

```diff
@@ -0,0 +1,11 @@
+#include "color2.h"
```

Comment:
Variant for color2 surface shader, color4, vec2 and vec4 variants also added. The name of the shader is "constant_" + MaterialX type. Existig shader can be used for color3 and vec3.